docs/build/guides/dapps/soroban-contract-init-template.mdx

```diff
@@ -1,442 +1,282 @@
 
-Stellar smart contracts are written in Rust, but suppose we want our frontend web application to work with these contracts: this is where templates come into the equation. Templates allow us to work with smart contracts in our frontend application. At its core a template consists of a frontend library and a script for interacting with the Stellar CLI.
+(You may wonder why it makes this unpopular choice. A fair question! The team wanted to balance actual-utility with broad approachability. Not everyone learning Stellar and Soroban is familiar with React, or any other UI library. It also demonstrates that core Soroban libraries all work with any JavaScript project.)
 
-Keep in mind you can use any frontend library to work with Stellar smart contracts. In this guide we'll use React to generate the template but it is worth noting that you can use whichever library you like or no library at all.
-
-We'll start off by installing the [React library](https://react.dev/) and then we'll walk through creating the initialization script, and finally wrap up by using the template to build, deploy, and generate contract bindings.
-
-## Creating the frontend
-
-The following command initializes a new Vite React project and installs the necessary dependencies:
+But note that the CLI's `contract init` command was made to work with _any_ frontend template. It need not even live on GitHub! Simply pass another valid git URL to the command, and it will clone that instead of the official repository:
 
 ```bash
-npm create vite@latest soroban-react-template --template react
-cd soroban-react-template
-npm install
-npm run dev
+# the following is NOT a valid git repository, and will not work 😄
+stellar contract init my-new-project --frontend-template https://whatever.cool/project/you/want
 ```
 
-## Creating the initialize script
-
-The initialization script is the heart of the contract template. This little but highly crucial script enables us to execute Stellar CLI commands which, in turn, enables us to build, deploy, and generate smart contract bindings.
+So how can you find other valid frontend templates?
 
-The initialization script can be written in bash script or JavaScript. This guide uses the latter but the same principles are involved when using bash scripts.
+At some point we may set up a DAO or other voting system to curate a list of high-quality templates, but for now... search GitHub! And GitLab. And any other source code site you know about.
 
-We start off by creating `initialize.js` files in our frontend project directory.
+In GitHub, in the main search bar, search for `"soroban-template-"`. With the quotes. Here's a direct link to the search results: [github.com/search?q=%22soroban-template-%22](https://github.com/search?q=%22soroban-template-%22)
```

Comment:
@ElliotFriend right now we can't search well for the templates, because I put the `astro` in the middle. This means you need to search for `soroban-*-template`, without quotes, and you get [a fair bit of noise](https://github.com/search?q=soroban-*-template). My bad! Would you be able to change the name of https://github.com/stellar/soroban-astro-template to `soroban-template-astro`? 

@CaptainPrinz could you also change the name of https://github.com/CaptainPrinz/soroban-react-template to `soroban-template-react`?

Reply:
@ElliotFriend! Just a quick tag for this.

Reply:
> @ElliotFriend ... Would you be able to change the name of https://github.com/stellar/soroban-astro-template to `soroban-template-astro`?

On it!

Reply:
@chadoh @briwylde08 rename is complete 👍🏻